extensions/rich_text_components/Skillreview/directives/skillreview.component.html

```diff
@@ -11,7 +11,7 @@
     split into multiple lines in the link text in the editor and shouldn't be
     removed.
   */
-  .cke_widget_oppiaskillreview {
+  .cke_widget_oppiaskillreview /deep/ {
```

Comment:
Mistake?

Reply:
Nope... need this to penetrate the view encapsulation done by angular (this would cause the styles to affect child components too. By default it doesn't.)

Reply:
Hmm, can we add a comment for this, this seems like quite an uncommon selector that is also quite hard to find and it seems that there are plans to deprecate it.